src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -644,7 +644,17 @@ void Pane::UpdateSettings(const TerminalSettings& settings, const GUID& profile)
         {
             // Update the parent of the control's settings object (and not the object itself) so
             // that any overrides made by the control don't get affected by the reload
-            _control.Settings().as<TerminalSettings>().SetParent(settings);
+            _control.Settings().as<TerminalSettings>().SetParent(settings.DefaultSettings());
+            if (settings.UnfocusedSettings())
+            {
+                // Update the control's unfocused appearance, and make sure to maintain the parentage
+                _control.UnfocusedAppearance(settings.UnfocusedSettings());
+                _control.UnfocusedAppearance().as<TerminalSettings>().SetParent(_control.Settings().as<TerminalSettings>());

```

Comment:
Whenever you see yourself doing stuff like...

```
x(y());
x().thing();
```

consider saving a local value to make it easier on yourself and the compiler (also, in the case of WinRT, for the runtime so it doesn't have to keep doing AddRef/Release/QueryInterface)

```
auto savedY{ y() };
x(savedY);
savedY.thing();
```

_don't call it `savedUnfocusedSettings`_

Reply:
In fact, I believe you can simplify this entire block...

```
auto savedY{ y() };
if (savedY) {
   savedY.SetParent()
}
_control.SetX(savedY);
```

because if you simplify the code, you see that it does this:

```
if (x != nullptr)
{
   x.DoThing();
   Save(x);
}
else
{
   Save(nullptr);
}
```

when it's totally safe to `Save(nullptr)`, there's no reason to check whether `x` _is nullptr_ before calling Save on it.